packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -719,12 +719,11 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
     }
 
     // Fetches a resolved Oracle price from the Oracle. Reverts if the Oracle hasn't resolved for this request.
-    function _getOracleEmergencyShutdownPrice() internal view returns (FixedPoint.Unsigned memory) {
+    function _getOraclePrice(uint256 requestedTime) internal view returns (FixedPoint.Unsigned memory) {
```

Comment:
Liquidatable needs access to this internal method to get the price for a specific liquidation time

Reply:
that's a good point. needs to get it at dispute time.